### PR TITLE
:movie_camera: record architectural decisions

### PR DESCRIPTION
**Pull request by beseven**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-133.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-133.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-133.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-133.dev.beta.nhschoices.net)

File: doc/architecture/decisions/0003-use-mongodb-for-search.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Date: 2017-02-27
+
+## Status
+
+Accepted
+
+## Context
+
+In order to perform a search on surgery name, a repository of GP Surgeries is needed.
```
Comment (neilbmclaughlin) [inline doc/architecture/decisions/0003-use-mongodb-for-search.md]: GP or surgery name

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-133.dev.beta.nhschoices.net)

**Comment by beseven** [comment]
@neilbmclaughlin, ready for re-review

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-133.dev.beta.nhschoices.net)

**Pull request merged**